packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,47 @@
+
+%global repo_orgname RedHatInsights
+%global repo_name insights-client-role
+%global repo_version 1.5
+
+%global role_orgname %{repo_orgname}
+%global role_name insights-client
+%global role_version %{repo_version}
+
+Name: ansiblerole-insights-client
+Summary: Packaging of the insights-client Ansible role
+Version: %{role_version}
```

Comment:
Normally you just define the version here and RPM makes a `%{version}` macro available for reference.